trie/database.go

```diff
@@ -372,8 +379,12 @@ func (db *Database) node(hash common.Hash) node {
 	db.lock.RUnlock()
 
 	if dirty != nil {
+		memcacheDirtyHitMeter.Mark(1)
+		memcacheCleanReadMeter.Mark(int64(dirty.size))
```

Comment:
Shouldn't it be this?
```suggestion
		memcacheDirtyReadMeter.Mark(int64(dirty.size))
```

Reply:
Damn, I though I fixed this.

Reply:
It's really weird, I found this issue on Friday and could have swarn I fixed it
apparently maybe I didn't commit

Reply:
Fixed.